pkg/platform/packet/packet.go

```diff
@@ -248,3 +245,51 @@ func (c *config) GetExpectedNodes() int {
 
 	return c.ControllerCount + workers
 }
+
+// Check cluster config is valid
+func (c *config) checkValidConfig() hcl.Diagnostics {
+	var diagnostics hcl.Diagnostics
+
+	diagnostics = append(diagnostics, c.checkNotEmptyWorkers()...)
+	diagnostics = append(diagnostics, c.checkWorkerPoolNamesUnique()...)
+
+	return diagnostics
+}
+
+// Make sure there's at least one worker pool
```

Comment:
```suggestion
// checkNotEmptyWorkers checks, if cluster has at least 1 node pool set.
```

All comments should end with dots, to resolve dispute if they should have a dot or not.

Reply:
Done.